manifests/server/grant.pp

```diff
@@ -103,8 +138,9 @@
   }
 
   $_unless = $unless_function ? {
-      false   => undef,
-      default => "SELECT 1 WHERE ${unless_function}('${role}',
+      false    => undef,
+      'custom' => $custom_unless,
+      default  => "SELECT 1 WHERE ${unless_function}('${role}',
```

Comment:
ack ack ack, didn't see `$custom_privilege` variable name, and thought it was assigned to `$_unless` as `$unless_function`